docs-api/frontity-packages/features-packages/smart-ads.md

```diff
@@ -0,0 +1,200 @@
+| **`library`** | string | yes      | The React component used to display the Ad. We can set this value to `"smartAdserver.SmartAd"` \(`SmartAd` component available in the `smartAdserver` package\). |
+| `priority`    | int    | no       | Assigns a priority in case more than one fill is assigned to that slot.                                                                                          |
+| **`props`**   | obj    | yes      | Props that will be passed to the `<Slot>` component _\(see table below\)_                                                                                        |
+
+#### The `props` property
+
+An object with props that will be passed to the `<Slot>` component.
+
+| Name           | Type   | Required | Description                                                                                                                                                                                                                                                       |
+| :------------- | :----- | :------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`siteId`**   | number | yes      | The `id` of the site as defined in [SmartAds](https://support.smartadserver.com/s/article/Ad-API-reference); represents a website or mobile application; [_more info_](https://support.smartadserver.com/s/article/Setting-up-inventory)                          |
+| **`pageId`**   | number | yes      | The `id` of the page as defined in [SmartAds](https://support.smartadserver.com/s/article/Ad-API-reference); represents a website or mobile application; [_more info_](https://support.smartadserver.com/s/article/Setting-up-inventory)                          |
+| **`formatId`** | number | yes      | The `id` of the format as defined in [SmartAds](https://support.smartadserver.com/s/article/Ad-API-reference); represents an ad slot on a page (medium rectangle, skyscraper...); [_more info_](https://support.smartadserver.com/s/article/Setting-up-inventory) |
+| **`callType`** | string | yes      | The type of the ad call. Possible values: `iframe` or `std`                                                                                                                                                                                                       |
+| **`tagId`**    | string | no       | The `id` of the container that will contain the ad. Default Value: `sas_${formatId}"`                                                                                                                                                                             |
+
+| **`width`** | number | no | The width of the ad. Used with callType `iframe`. |
```

Comment:
The table here is broken, there is an new line character that should be removed (line 91).

```suggestion
| **`tagId`**    | string | no       | The `id` of the container that will contain the ad. Default Value: `sas_${formatId}"`                                                                                                                                                                             |
| **`width`** | number | no | The width of the ad. Used with callType `iframe`. |
```